change/react-native-windows-2019-08-16-14-49-31-jsi-engine_override.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "none",
+  "comment": "Add jsiEngineOverride option to devsettings",
+  "packageName": "react-native-windows",
+  "email": "tudorm@microsoft.com",
+  "commit": "086aba4eb8a1e1ab0b80c212290a541bf63c2ba9",
+  "date": "2019-08-16T21:49:31.769Z"
+}
```

Comment:
In codeflow I see the the runtimeholder files were moved from vnext/ReactWindowsCore, but I don't see them being deleted from vnext/ReactUWP/base/... . Not sure if this is just the way the tooling works.

Reply:
On GitHub they do show up as "move + modify", probably that confuses CodeFlow: [https://github.com/microsoft/react-native-windows/pull/2944/files](https://github.com/microsoft/react-native-windows/pull/2944/files)

---
In reply to: [314905668](https://github.com/microsoft/react-native-windows/pull/2944#discussion_r314905668) [](ancestors = 314905668)